include/inviwo/core/interaction/events/interactionevent.h

```diff
@@ -49,9 +51,15 @@ class IVW_CORE_API InteractionEvent : public Event {
     KeyModifiers modifiers() const;
     void setModifiers(KeyModifiers modifiers);
     std::string modifierNames() const;
+    
+
+    using ToolTipCallback = std::function<void(std::string)>;
+    void setToolTipCallback(ToolTipCallback callback);
```

Comment:
One uses the 'getToolTipCallback' like: 
```
pickingEvent->getToolTipCallback()("Tool Tip String");
```

From a Processor Developer point of view that syntax will be quite confusing because you want to "set the tooltip message". Would make more sense if one would call something like the following in the picking callback: 
```
pickingEvent->setToolTip("Tool Tip String");
```